testing/py3-empy/APKBUILD

```
@@ -0,0 +1,29 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=py3-empy
+_pkgname=empy
+pkgver=3.3.4
+pkgrel=0
+pkgdesc="A powerful and robust templating system for Python."
+url="http://www.alcyone.com/software/empy/"
+arch="noarch"
+license="LGPL-2.1-or-later"
+depends="python3"
+source="http://www.alcyone.com/software/empy/empy-$pkgver.tar.gz
+        10-fix-test-case.patch"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+  python3 setup.py build
```

```suggestion
    python3 setup.py build
```
The indentation should be a tab instead of spaces. Please fix build, check & package.

---

In another APKBUILD PR I was advised to use spaces over tabs. @maxice8  ?

---

i can't remember of the top of my head, i probably meant to align likes after `\` with spaces

---

Maybe I imagined it.

Maybe a style guide on the wiki where all these things are documented would be helpful.

---

Fixed.